meerkat_frontend/src/files/pages/contact_tracing.html

```diff
@@ -32,8 +32,8 @@
                         <p id="box_heading"></p>
                         <div class="box__type-selector pull-right">
                             <a href="" onclick="exportTableToXLS('contact-summary-table','contacts');return false;" class="csv">
-								Excel
-							</a>
+                                Excel
+							              </a>
```

Comment:
Issue here is use of tab character. Lets move away from tab characters to hard spaces